lib/usd/pxrUsdPreviewSurface/CMakeLists.txt

```diff
@@ -68,13 +68,14 @@ target_include_directories(${TARGET_NAME}
 # -----------------------------------------------------------------------------
 # link libraries
 # -----------------------------------------------------------------------------
-target_link_libraries(${TARGET_NAME} 
+target_link_libraries(${TARGET_NAME}
+    PUBLIC
+        tf
```

Comment:
Declare a transitive dependency to tf, which has the transitive dependency to Boost that is required to build pxrUsdPreviewSurface plugin with USD 19.11.